configs/common/packages.yaml

```diff
@@ -30,6 +30,9 @@ packages:
     - any_of: ['@2.3.0 ~openmp']
       when: '%intel@2021:'
       message: '2.3.0 is the last version to use C++17, use with Intel Classic'
+    - any_of: ['@2.3.0 ~openmp']
```

Comment:
This change is already included in https://github.com/JCSDA/spack-stack/pull/1464 and will no longer show up once #1464 is merged and this PR is updated from develop